src/components/App.js

```diff
@@ -1,8 +1,196 @@
+import React, { useState } from 'react';
+import SelectNumber from './Select';
+import Board from './Board';
+import { alphabet } from '../letters/Alphabet';
+import { wordDictionary2 } from '../letters/WordDictionary2';
+import { wordDictionary3 } from '../letters/WordDictionary3';
+import { wordDictionary4 } from '../letters/WordDictionary4';
+import { wordDictionary5 } from '../letters/WordDictionary5';
+import { letterPoints } from '../letters/LetterPoints';
 
 function App() {
+  const [selectNumber, setSelectNumber] = useState(false);
+  const [wordNo, setWordNo] = useState(1);
+  const [numberSelected, setNumberSelected] = useState('');
+  const [squares, setSquares] = useState([]);
+  const [wordLengths, setWordLengths] = useState([]);
+  const [remainingAlphabet, setRemainingAlphabet] = useState(alphabet);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [score, setScore] = useState(0);
+  let game;
+  let words = [];
```

Comment:
Looks like this is made for `handleClick`? But I'm not sure why its defined outside of the fn

Reply:
It was moved out of the function as I was using it in the HTML. I'll move it back in and come up with another solution for the HTML as you didn't like that code either.